### ISSUE TITLE
Add license to pom file

### ISSUE THREAD

**Issue by benmccann** [haraldk/TwelveMonkeys#213]
Thank you for licensing this software under the BSD license. We have tools which automatically report the license of libraries that we use by looking in the pom file.

To see an example of a BSD licensed library being reported in a pom file:
https://repo1.maven.org/maven2/org/antlr/antlr4-master/4.5.1-1/antlr4-master-4.5.1-1.pom

**Comment by haraldk** [comment]
Hi Ben,

Thanks for reporting.

Do you know if it's enough to add it to the "parent" (top-level) POM, or do the license have to be added to each POM individually?

Feel free to submit a pull-request, otherwise I'll look into it for the next release.


Regards,

-- 
Harald K

**Comment by benmccann** [comment]
Yes, it's common practice to add it only to the parent

**Comment by haraldk** [comment]
Thanks!

I've now added the following to the parent POM:

```
    <licenses>
        <license>
            <name>The BSD License</name>
            <url>https://github.com/haraldk/TwelveMonkeys#license</url>
            <distribution>repo</distribution>
        </license>
    </licenses>
```

Will be released as part of the 3.3 release.

-- 
Harald K

**Comment by benmccann** [comment]
Thanks!
